man/nvim.1

```diff
@@ -140,7 +136,7 @@ Sets the options 'fkmap' and 'rightleft'.
 Hebrew mode.
 Sets the options 'hkmap' and 'rightleft'.
 .It Fl V Ns Oo Ar N Oc Ns Op Ar file
-Verbose.
+Be verbose.
```

Comment:
suggest "Verbose mode", it has parallel form with "Debugging mode" and "Recovery mode".